components/ingest-service/pipeline/chef_run.go

```diff
@@ -35,7 +36,8 @@ func NewChefRunPipeline(client backend.Client, authzClient iam_v2.ProjectsClient
 		processor.BuildRunProjectTagger(authzClient),
 		publisher.BuildNodeManagerPublisher(nodeMgrClient),
 		processor.BuildRunMsgToBulkRequestTransformer(client),
-		publisher.BuildBulkRunPublisher(client, maxNumberOfBundledRunMsgs),
+		publisher.BuildMsgDistributor(publisher.BuildBulkRunPublisher(
```

Comment:
Super nit but, I can read this better if you ident it this way:
```go
publisher.BuildMsgDistributor(
  publisher.BuildBulkRunPublisher(client, maxNumberOfBundledRunMsgs),
  numberOfParallelBulkPublishers,
  maxNumberOfBundledRunMsgs,
),
```

Reply:
That does look better. Thanks.